src/main/java/uk/gov/pay/ledger/transaction/model/TransactionFactory.java

```diff
@@ -138,6 +144,7 @@ private Transaction createRefund(TransactionEntity entity) {
                     .withParentExternalId(entity.getParentExternalId())
                     .withGatewayPayoutId(entity.getGatewayPayoutId())
                     .withPaymentDetails(paymentDetails)
+                    .withSettlementSummary(settlementSummary)
```

Comment:
What will be the behaviour when paidout_date is not available ? do we get `settlement_summary` without any fields ?